### DISCUSSION TITLE
Set default glide "Quality"

### DISCUSSION THREAD

**Discussion by schwartzmj** [spatie/statamic-responsive-images#115] [Q&A] [answered]
Apologies if I missed something in the docs. Is there a way to set/override the default "Quality" setting for the Glide images that this package generates (without doing it on every tag)?

Thank you!

**Comment by riasvdv** [comment, marked as answer]
No, there is currently no way to set default glide parameters

**Comment by ncla** [comment]
Hey @schwartzmj, it is now possible to override quality settings in config file, you can also override in front-end templates for each breakpoint individually! This feature was added in `v2.12.0 - 2022-05-23` release.

**Reply by schwartzmj** [reply]
Awesome! Thank you for letting me know!
